variables-extensions.tf

```diff
@@ -95,7 +121,7 @@ variable "metrics_server_helm_values_files" {
 
 # Cluster autoscaler
 
-variable "cluster_autoscaler_enabled" {
+variable "cluster_autoscaler_install" {
```

Comment:
I thought we supported autoscaler?

Reply:
Yeah, this is just fixing the names you brought up in an earlier PR.